client/src/components/pages/Home/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
+import './index.css';
+
+const HomePage = () => {
+  return (
+    <div className="home">
+      <div className="home__header">
+        <div className="home__header__img__overlay"></div>
+      </div>
+      <div className="about">
+        <h3 className="about__title">About</h3>
+        <div className="about__block">
+          <div className="about__block__text">
```

Comment:
rename the class to follow the BEM notation: `about__text`, `about__content`, `about__main-content` or something similar